install.sh

```diff
@@ -84,11 +100,18 @@ if [ "$ext_url" = "" ]; then
 fi
 checksum_url="${EXTENSION_CHECKSUM_URL:-}"
 download_max_sec="${MAX_DOWNLOAD_SEC:-30}"
+
 ext_filename=$(basename -- "$ext_url")
 download_dir=`mktemp -d -t extension-XXXXXX`
 ext_file="$download_dir/$ext_filename"
 if [ -f $ext_file ]; then
     rm $ext_file
 fi
+
+ext_vars=$(echo "$EXTENSION_EXT_JS_VARS" | jq -c '.')
+ext_js_file_path="${EXTENSION_EXT_JS_FILE_PATH:-}"
```

Comment:
it seems that this variable is unused. Please cleanup

Reply:
cleaned up, tks